scss/main.scss

```diff
@@ -86,6 +86,22 @@ $animation-length: 80s;
   }
 }
 
+.logo {
+  background: linear-gradient(90deg, #00FFFF 0%, #AA96CA 46.35%, #FF6B98 100%);
+  width: 50px;
+  height: 50px;
+  border-radius: 50px;
+}
+
+.navbar-brand {
```

Comment:
the brand logo messes up the centering of the links - lets add `position: absolute` to this class